modules/core/src/main/java/org/apache/ignite/internal/visor/node/VisorDataStorageConfiguration.java

```diff
@@ -460,6 +472,7 @@ public boolean isWalCompactionEnabled() {
         metricsSubIntervalCount = in.readInt();
         metricsRateTimeInterval = in.readLong();
         walAutoArchiveAfterInactivity = in.readLong();
+        walForceArchiveTimeout = in.readLong();
```

Comment:
Should we provide backward compatibility here?

Reply:
Ignite doesn't provide backward compatibility for this.
So I think no, we shouldn't handle backward compatibility here.